docs/pages/build/internal-distribution.md

```diff
@@ -126,8 +126,6 @@ Press the "Install" button on the build page and follow the instructions present
 
 It's possible to run internal distribution builds non-interactively in CI using the `--non-interactive` flag; however, if you are using ad hoc provisioning on iOS you will not be able to add new devices to your provisioning profile when using this flag. After registering a device through `eas device:create`, you need to run `eas build` interactively and authenticate with Apple in order for EAS to add the device to your provisioning profile. [Learn more about triggering builds from CI](/build/building-on-ci.md).
 
-<!--
-(@dsokal) this is not implemented yet
-
+{/* (@dsokal) this is not implemented yet
```

Comment:
The new `{/* */} comment system does not render correctly on the page and does not hide the comments.

For example: 
<img width="910" alt="CleanShot 2022-09-06 at 03 07 45@2x" src="https://user-images.githubusercontent.com/10234615/188514529-8b93b1e5-e9d6-40c4-bc70-08b5702671d6.png">

<img width="775" alt="CleanShot 2022-09-06 at 03 07 28@2x" src="https://user-images.githubusercontent.com/10234615/188514554-1020a663-5369-4088-b85c-5ae2bf3c5df7.png">

Is there any other configuration required to set this up and render it correctly? Or do we have to switch to `.mdx`?

Reply:
This means that I need to apply those changes within the MDX update PR, thanks for catching this! 🙏

Reply:
No worries. Let me know if I can help with anything.